lib/jwt_signed_request/middlewares/faraday.rb

```diff
@@ -29,24 +30,14 @@ def call(env)
 
       private
 
-      attr_reader :app, :env, :options, :jwt_token
+      attr_reader :app, :env, :bearer_schema, :options, :jwt_token
 
       def authorization_header
         bearer_schema? ? "Bearer #{jwt_token}" : jwt_token
       end
 
       def bearer_schema?
-        options[:bearer_schema] == true
-      end
-
-      def optional_settings
-        {
-          secret_key:                 options[:secret_key],
-          algorithm:                  options[:algorithm],
-          additional_headers_to_sign: options[:additional_headers_to_sign],
-          key_id:                     options[:key_id],
-          issuer:                     options[:issuer],
-        }.reject { |_, value| value.nil? }
```

Comment:
To simplify the code, nil options are no longer removed here. They default to nil anyway in `Verify` and `Sign` (ditto in rack middlware).